### ISSUE TITLE
Mega-Issue for "GreiferBait" mechanics that should be temporarily (or permanently) nerfed/changed for the PCTide

### ISSUE THREAD

**Issue by Jezithyr** [space-wizards/space-station-14#28606]
This issue is specifically for mechanics/items that cause issues *when used by a nefarious player*, mechanics/items that result in issues from lack of player understanding (ie: singuloose) should be tracked in this issue instead: #28604

This is a list of mechanics/changes that should be made to considerably decrease the amount of damage a greifer can do without requiring major changes. These changes should be fast/easy to fix without requiring major changes to design or code. These changes should be tracked and any PRs should reference this issue so that they can be reverted after the Tide wanes.

If a specific change turns out to improve the game without any downsides we might keep it in, otherwise the changes are temporary and *will be reverted*.

In the future, we should make make any temporary nerfs configurable via CVAR(s), for example: Enable/Disable "Greifable Mechanics"

If you have any suggestions for other mechanics/gameplay that is problematic feel free to suggest it in the discussion.

Please keep actual discussion to a minimum or use #contributors in discord, the discussion below should only be used for listing mechanics/gameplay that are problematic to new players.

Specifically the goal with this is to make it more difficult to obtain items or use mechanics that have *high potential or power* to cause large amounts of damage when in the hands of a griefer, while still keeping them in the game/playable for the regular use-case.

Problematic gameplay/mechanics:
- [ ] Nerf clf3 
- [x] Welderbombing

**Comment by Yousifb26** [comment]
I believe clf3 already received a nerf upstream that recently got merged here, it now requires heat to produce it

**Comment by Jezithyr** [comment]
> I believe clf3 already received nerf, it now requires heat to produce it

I don't know if that is enough to mitigate its ease of creation and the effect it has when used. I'm not familar with how clf3 is balanced atm but someone mentioned that it might still be problematic which is why I included it on this list for a possible (temporary) nerf.

With the upcoming surge in player numbers we're going to have alot higher of an player to admin ratio so some griefers may slip through the cracks. The goal with any temporary nerfs we make is to remove/mitigate the mechanics/items a greifer/raider might use to derail/ruin a round for the whole station. Is it going to ruin solo/greytide gameplay, yeah definitely but that's a far better outcome than having an entire round ruined because a greifer was spamming clf3 bombs and admin who would normally ban them after the first bomb was busy dealing with raiders on another server. 

So the plan is to nerf items/mechanics that give individual players the means to cause widespread damage during the large player influx, and then revert the nerfs after the initial wave of players passes and admins have better server coverage again.

**Comment by Hrosts** [comment]
Are there currently any protections against building walls/traps at arrivals/cryo?

**Comment by thebadman4662** [comment]
People are not robust enough to even use current CLF3 vs nukies and griefers are way less robust than normal crew, especially if they were not playing SS14 in first place

**Comment by ninruB** [comment]
Electrifying doors/vendors, maints grilles, etc. should receive some sort of nerf during the pctide.

**Comment by Hmeister-real** [comment]
> griefers are way less robust than normal crew, especially if they were not playing SS14 in first place

Not true. In a lot of cases, i see Russian griefers being robust in combat, or doing non-common knowledge stuff like making maxcaps. I haven't played ss14 for the freeze, waiting for newmed to start playing again, so maybe it's different now, but i doubt it.

**Comment by lexs0sa** [comment]
Hi, long time player, short time ban evader/griefer here, CLF3 is nerfed to not make reliably for a griefer, this definitely shows how out of touch you are and unprepared for the inevitable griefing that my Russian brothers will provide the community. 

Welding bomb and CLF3 were nerfed to be more funny than viable for griefing, but maybe 6 months ago this would have been a good idea. Good luck in the next coming weeks, "Girls"!

**Comment by lexs0sa** [comment]
Oh Jez I found your f-list by the way, you are gonna be famous, say cheese! Glad you don't have "Parent and Child Play" like DmugLeaf does on your page.

**Comment by gostt899** [comment]
Should probably prevent the smoke and foam reaction from mixing with other chems temporarily as they can be extremely lethal to a large crowd like bars and medical.

**Comment by thebadman4662** [comment]
> Should probably prevent the smoke and foam reaction from mixing with other chems temporarily as they can be extremely lethal to a large crowd like bars and medical.

Uh, are we trying to remove all weapons from revolutionaries in this issue thread or only tider tier mass destruction like welderbombing? Cause if latter then its chemistry/cargo locked, in theory at least and will require some effort and knowledge.

**Comment by thebadman4662** [comment]
Perhaps APC deconstruction should be made harder.

Partially because how skill issued your average engi is, partially because how easy it is to do.

Screwdrive APC, crowbar APC. Boom power is gone in about 5 seconds. Repeat till half or more of station is dark.

I do not even think removing APC hust is hard but would need to test if it needed lengthy welder doafter to cover up your naughty griefer tracks...


Similiar to this, its mega easy to mass destroy station lights with something as simple as a crowbar, but at least flashlights still exists and station can still function, albeit worse.

**Comment by Quantum-cross** [comment]
> Perhaps APC deconstruction should be made harder.

Absolutely agree, there needs to be wire panels with power wires to cut. Do APCs log access? If not, it should so that logprobe can read this. Additionally, this would allow AIA, ACC, and LOG wires to be relevant.

Also screwdriver + crowbar often trivializes "apc power off station events" as 3 simple actions will re-turn on an APC without any access or insuls.

I will create a separate issue for this if I can't find one.

**Comment by lzk228** [comment]
PCTide :trollface:

**Comment by slarticodefast** [comment]
Closing since the PC tide is over.
